cmake/Dependencies.cmake

```diff
@@ -1,6 +1,25 @@
+set(UBSAN_FLAG "-fsanitize=undefined")
+
+macro(disable_ubsan)
```

Comment:
Needs an explanation of why this is necessary (we use it to disable protobuf ubsan)